third_party/pdks/skywater/skywater130/pdk/v0_0_2/apr/tapcell.tcl

```diff
@@ -0,0 +1,4 @@
+tapcell \
+  -endcap_cpp "2" \
```

Comment:
-endcap_cpp is not needed

Reply:
This was taken directly from OR. Should it be fixed there as well?
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/platforms/sky130hd/tapcell.tcl

Reply:
yes, it should

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/388

Reply:
:+1: